internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -143,3 +145,67 @@ func TestCRDHaveValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestOwnedCRDsForResources(t *testing.T) {
```

Comment:
```suggestion
func TestCheckResources(t *testing.T) {
```

By default, it is Test<NameFunc> that will be checked and then, we can increase the test by adding further scenarios if required.